draft-ietf-mls-architecture.md

```diff
@@ -213,52 +213,52 @@ Clients may wish to do the following:
 
  -  receive a message from someone in the group.
 
-At the cryptographic level, Clients in groups (and by extension Members)
-are peers. For instance, any Client can add a member to a group. This
+At the cryptographic level, clients in groups (and by extension Members)
+are peers. For instance, any client can add another client to a group. This
 is in contrast to some designs in which there is a single group
 controller who can modify the group. MLS is compatible with having
 group administration restricted to certain users, but we assume that
-those restrictions are enforced by authentication and access control.
+those restrictions are enforced by authentication and access control
+at the application layer.
 Thus, for instance, while it might be technically possible for any
-member to send a message adding a new member to a group, the group
+member to send a message adding a new client to a group, the group
 might have the policy that only certain members are allowed to make
 changes and thus other members can ignore or reject such a message
 from an unauthorized user.
 
 ## Group, Members and Clients
 
-In MLS a Group is defined as a set of Members who possibly use multiple
-endpoint devices (Clients) to interact with the Messaging Service.
-These Clients will typically correspond to end-user devices such as phones,
-web clients or other devices running MLS.
+Informally, a group is a set of users who possibly use multiple endpoint
+devices to interact with the Messaging Service.
+These members will typically correspond to end-user devices such as phones,
+web clients or other devices running MLS, which are called clients.
 
-Each member device owns a long term identity key pair that uniquely defines
-its identity to other Members of the Group.
-Because a single Member may operate multiple devices simultaneously
+Each client owns a long term identity key pair that uniquely defines
+its identity to other clients or members a the Group.
+Because a single user may operate multiple devices simultaneously
 (e.g., a desktop and a phone) or sequentially (e.g., replacing
-one phone with another), the formal definition of a Group in MLS
-is the set of Clients that has legitimate knowledge of the shared (Encryption)
-Group Key established in the group key establishment phase of the protocol.
+one phone with another), the formal definition of a group in MLS
+is the set of clients that has knowledge of the shared group secret
+established in the group key establishment phase of the protocol.
 
```

Comment:
To account for both multi-device modes we discussed, I'd like to add the following:

"Multiple user devices can be grouped, appearing as one virtual client to the rest of the group."